src/components/Utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+export function normalizeItem(String) {
+  return String.trim()
```

Comment:
Here's you're passing `String` as an argument, and this works, but, `String` is already an [Object declared by Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) 

Let's change this to something different.

I'd suggest you changing this to `item` or `itemName`.

```js
// Something like this should work.
export function normalizaItem(item) { ... }
```